cla_public/templates/macros/subform.html

```diff
@@ -8,24 +8,21 @@
         WTForm field
 #}
 {% macro callback(form) %}
-  {% call Form.fieldset(
-    is_subfield=True,
-    controlled_by=form.contact_type,
-    control_value="callback",
-    use_row=False,
-    persist_values=True
-  ) %}
-    <div class="form-notice" role="alert">
+  {% call Form.group(None,
+      class_='form-subfield',
+      use_row=False,
+      controlled_by=form.contact_type,
+      control_value='callback') %}
+    <div class="form-group form-notice" role="alert">
       <ul class="styled-list">
```

Comment:
there are supposed to be 2 separate lists of notes, one for "call me back" and a different one for "call someone else instead of me"